pkg/master/master.go

```diff
@@ -124,6 +125,9 @@ type Config struct {
 	// Used to start and monitor tunneling
 	Tunneler genericapiserver.Tunneler
 
+	// Enables using master leases
+	EnableLeases bool
```

Comment:
Sorry its not clear to me how adding this to pkg/master/ instead of adding it to pkg/genericapiserver/ reduces the delta?